microhapulator/data/marker_details_template.html

```diff
@@ -97,7 +99,9 @@ <h4>Read Mapping</h4>
                     <em>expect</em> to see with perfectly even coverage across all markers: that is, a "Mapped Rate" of
                     > 1.0 indicates higher than expected coverage and < 1.0 indicates lower than expected coverage. "Off
                     Target Reads" and "Off Target Rate" refer to the number and percentage of reads that map to the
-                    marker reference, but preferentially map elsewhere when aligned to the entire genome.
+                    marker reference, but preferentially map elsewhere when aligned to the entire genome. <strong>N/A
+                    values in these columns indicate that the marker definition file does not have GRCh38 coordinates
+                    for all SNPs, which are required for off-target analysis.</strong>
```

Comment:
Including a warning in the report, in the likely event that the terminal warning is missed.